### PR TITLE
[CRIMAPP-421] Add PSE as a competency

### PR DESCRIPTION
**Pull request by hiboabd**
## Description of change
Supervisors can now assign a PSE competency to a caseworker
A caseworker with a PSE competency can pick up a PSE application via the get next button
A caseworker without a PSE competency can’t pick up a PSE application via the get next button

## Link to relevant ticket
[CRIMAPP-421](https://dsdmoj.atlassian.net/browse/CRIMAPP-421)

## Notes for reviewer

This is a change to the way work queues and competency assignment currently works in comparison to eForms.  
Work needs to be done consider how to explain how this works to supervisors and caseworkers on review and ensure they are trained to understand the differences

### After changes:

**Changes to the competency form** 
Screenshot 2024-01-29 at 11 49 54
Screenshot 2024-01-29 at 11 49 59

**Retrieving a PSE application as a caseworker with the PSE competency** 

Screenshot 2024-01-29 at 11 37 55
Screenshot 2024-01-29 at 11 37 41

**Retrieving a PSE application as a caseworker without the PSE competency** 

Screenshot 2024-01-29 at 11 38 08

## How to manually test the feature

**Note: Requires a pse application to have been submitted** 
**Note: Requires a supervisor user and a caseworker user** 

Assigning a PSE competency 
- Log into review as a supervisor. Click on manage competencies. Go to a caseworker click their competencies to go to the competency form 
- Confirm that you can see post submission evidence listed as a competency option 
- Confirm that you can assign the caseworker the post submission evidence competency (Note: the caseworker will need at minimum both the CAT1 and PSE competency to be able to work on PSE applications)

Picking up a PSE application via the Get next button 
- Log into review as the caseworker you have just assigned the PSE competency to. 
- Go to the your list tab and click get next and confirm that you can get the next available PSE. (Note: you may have other older applications that are assigned first)

[CRIMAPP-421]: https://dsdmoj.atlassian.net/browse/CRIMAPP-421?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: config/locales/en/casework.yml [hunk 1]
```diff
@@ -118,7 +118,7 @@ en:
     overall_offence_class: Overall offence class
     overview: Overview
     partner: Partner
-    post_submission_evidence: Post submission evidence
+    post_submission_evidence: PSE
```
Comment (arthurashman) [inline config/locales/en/casework.yml:121]: Are we happy / have we checked that they are all good with 'PSE'?

**Review by timpeat — APPROVED** [review]

**Comment by hiboabd** [comment]
> Looks great. All working well for me. I assume it's ok that I can assign a PSE to myself from the open apps table? (not using get next)
> 
> Question from me: Do we need to check if there is a user need to build somone's queue to be ONLY PSE? (i.e. no CAT1 initials, just CAT1 PSE)

Thanks Arthur, yes we've left in the ability to assign a PSE to yourself via the assign button. On your second question, I don't think that's a user need

File: config/locales/en/casework.yml (hunk 1 above)
Comment (hiboabd) [reply config/locales/en/casework.yml:121]: Nope haven't checked with anyone. Do you mean ucd or users or both?

Comment (arthurashman) [reply config/locales/en/casework.yml:121]: Ultimately users, but UCD might have a steer for now. Not a big deal - I just wanted to ask to make sure it's a term everyone is going to understand. As a general rule we should be steering away from acronyms where possible

**Pull request merged**